index.php

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <?php $base = "../../" ?>
+    <base href="../../">
+    <script src="js/jquery-2.2.4.min.js"></script>
+    <script src="js/facebox.js"></script>
+    <script src="js/gameSettings.js"></script>
+    <link rel="stylesheet" type="text/css" href="css/facebox.css"/>
+    <link rel="stylesheet" type="text/css" href="css/main.css"/>
+    <link rel="stylesheet" type="text/css" href="css/bootstrap.css"/>
+    <script type="text/javascript">
+        jQuery(document).ready(function($) {
```

Comment:
can you find out the point of this?

Reply:
This script is used for creating the "Play Now!" button at the bottom of the page. The style sheets are used for making the page look similar to the other Dr. Ecco pages.